libc/src/__support/HashTable/generic/bitmask_impl.inc

```diff
@@ -7,37 +7,46 @@
 //===----------------------------------------------------------------------===//
 
 #include "src/__support/endian.h"
+#include "src/__support/common.h"
 
 namespace LIBC_NAMESPACE {
 namespace internal {
+
+// FIXME: The GPU build cannot use 'uintptr_t' directly.
+#ifdef LIBC_TARGET_ARCH_IS_GPU
+  using bitmask_t = uint64_t;
```

Comment:
Do these GPU architectures of interest always have 64-bit general registers?

Reply:
Here it is better to make it of register width such that each scan only involves one register.

Reply:
Alright, that would require divergence from `intptr_t` anyway so I guess this patch would still be necessary for that behavior.

Reply:
Yeah. Good to know that on GPU `uintptr_t/size_t` does not resolve to register-wide types.